### ISSUE TITLE
Which argument not passed to geom_alignment when within function call

### ISSUE THREAD

**Issue by ewallace** [tengfei/ggbio#67]
```{r }
# data is taken from 
# http://faculty.ucr.edu/~tgirke/HTML_Presentations/Manuals/Rgraphics/Rgraphics.pdf
# http://faculty.ucr.edu/~tgirke/HTML_Presentations/Manuals/Workshop_Dec_12_16_2013/Rgraphics/data.zip

# load transcript annotation as txdb
txdb <- makeTxDbFromGFF(file="data/TAIR10_GFF3_trunc.gff", format="gff3")

# define region of interest:
wh <- GRanges("Chr5", IRanges(4000, 8000))

# plot with ggbio
ggplot(data=txdb) + geom_alignment(which=wh), names.expr = "gene_id")

# define function to plot subregion with default theme etc
plot_subtxdb <- function(mytxdb,mywhich) {
    ggplot(data=mytxdb) + 
        geom_alignment(which=mywhich, names.expr = "gene_id")
}

# Attempt to run function
plot_subtxdb(mytxdb=txdb,mywhich=wh)

# I get:
# Error in .local(obj, ...) : object 'mywhich' not found
```

**Comment by lawremi** [comment]
Attempted a fix to come with version 1.18.4.

**Comment by ewallace** [comment]
Thanks, but the fix didn't work in ggbio 1.18.4

**Comment by lawremi** [comment]
Sorry about assuming I had fixed it. ggbio is crazier than I thought. Pretty sure things work better in 1.18.5.
